tech.json

```diff
@@ -572,7 +572,14 @@
             {
               "name": "canSnailClimb",
               "requires": [ "canUseEnemies" ],
-              "note": "Piggybacking on top of a climbing snail to get up a wall"
+              "note": [
+                "The ability to repeatedly manipulate Yards (snails) and use them as platforms.",
+                "The snail is damaging when out of its shell so Samus must face the center of the snail to cause it to hide.",
+                "The snail climb consists of Samus jumping away from the snail as it becomes active, then turning around to return to the snail and resposition for the next jump.",
+                "Standing on the edge of the snail and turning around underwater does carry the risk of falling off.",
+                "SpringBall can bounce on an active snail (only one bounce when underwater without HiJump)."
```

Comment:
Might be worth mentioning that this does not count for simply using a snail as a platform one time